src/System.Text.Json/src/System.Text.Json.csproj

```diff
@@ -66,4 +66,17 @@
   <ItemGroup>
     <ReferenceFromRuntime Include="System.Private.CoreLib" />
   </ItemGroup>
+
+  <Target Name="EnsureResourceDirectoryExists" BeforeTargets="_BatchGenerateResxSource">
```

Comment:
This looks like a bug in _BatchGenerateResxSource, can you just fix that?  FWIW you should never hit this in a normal build since the package build comes after binary build so I wouldn't bother putting the workaround here if you plan on putting a better fix in arcade.

Reply:
Ok, will remove.